### PR TITLE
Test of HTML table to record insturtors and hosts

### PR DESCRIPTION
**Pull request by rkdarst**
- This is done with HTML tags, instead of markdown syntax, since the
  markdown syntax puts an entire row on one line - horrible for
  conflicts and merging!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,107 @@
...
+difficult problems, especially configuration problems at the very
+beginning of each day.  Optional.  **Helper:** breakout room helper, listed at
+end (not per day) since they are usually expected to be a helper every
+day. **(name)** in parentheses: "I offer to do it
+but I am happy if someone replaces me here."
+
+Overlapping roles are allowed when you think it's reasonable!
+
+
+<table>
```
Comment (blankdots) [inline README.md:20]: wouldn't it be ok to use markdown table instead of html, it is less verbose ?

**Comment by rkdarst** [comment]
From what I could tell, everything would be on fever lines, meaning very
high likelihood of conflicts and much harder merging. I designed this to be
easy for PRs.  And didn't see anything like rst's list-table...

What do you think?

**Review by wikfeldt — APPROVED** [review]
i agree html table is better to minimize risk of conflicts

**Comment by wikfeldt** [comment]
we can at least start with this and change later if any improvements are found

**Pull request merged**